code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -15,6 +15,7 @@
 	var/opened = FALSE
 	var/welded = FALSE
 	var/locked = FALSE
+	var/speciallock = FALSE // var that can be used for special ways to open

```

Comment:
Instead of implementing a new variable with cryptic description, just override can_open on large crates to always return FALSE.

Reply:
Alternatively, use a bitflag to decide how crates can be opened.